cmds/grpc-backend/main.go

```diff
@@ -173,6 +175,14 @@ func main() {
 		ctx    = context.Background()
 		logger = logging.WithValuesFromContext(ctx, logging.Logger)
 	)
+	if *profServiceName != "" {
+		if err := profiler.Start(
+			profiler.Config{
+				Service:        *profServiceName,
+				ServiceVersion: "v1"}); err != nil {
```

Comment:
Seems like we should factor out the service version into a named constant.

Reply:
Done.